tasks/run-matlab-command/v0/main.ts

```diff
@@ -11,13 +11,19 @@ async function run() {
     try {
         taskLib.setResourcePath(path.join( __dirname, "task.json"));
         const command = taskLib.getInput("command", true);
-        await runCommand(command as string);
+        const startupopts = taskLib.getInput("startup-options");
```

Comment:
Can we do camelCase here too to be consistent with the rest of the code? Ignore the other comment I had for a code change. I'm sure there's some fancy typescript thing we can do but it wasn't working how I wanted, and I think this is easy enough to understand.

Reply:
Fixed